dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/entity/TaskDefinition.java

```diff
@@ -218,6 +218,12 @@ public class TaskDefinition {
      */
     private TaskExecuteType taskExecuteType;
 
+    /**
+     * process task related list
+     */
+    @TableField(exist = false)
+    private List<ProcessTaskRelation> processTaskRelationList;
```

Comment:
I think it's better to use `TaskDefinitionVo` and don't add these fileds in dao entity.